### ISSUE TITLE
where have you modified from original 3D GS？

### ISSUE THREAD

**Issue by Erhushenshou** [ubc-vision/3dgs-mcmc#27]
Have you modified the cuda kernel？ Or did you only modified the train.py and gaussian_model.py？

**Comment by shakibakh** [comment]
Yes the cuda code has also been modified. Also, other files have been added/edited. I suggest you check the diff to see the edits. For the cuda code, you need to clone the repo with --recursive or download "diff-gaussian-rasterization" submodule from the url and branch that is provided in .gitmodules.
